src/docs/codelabs/extended-flex-widget.md

```diff
@@ -0,0 +1,597 @@
+
+  For your Column, set the:
+  </li>
+ 
+  <li markdown="1">`mainAxisSize` property to `MainAxisSize.min`
+  </li>
+  <li markdown="1">`crossAxisAlignment` property to `CrossAxisAlignment.start`.
+  </li>
+</ul>
+    
+{{site.alert.end}}
+<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=30ccbe0fcf31cc10eafba3aea8ff0697&amp;theme=dark&amp;split=60" width="100%" height="400px"></iframe>
+
+#### Exercise: Wrap the Column in a Row
+{:.no_toc}
+{{site.alert.secondary}}
+  
+
+  Wrap your `Column` in a `Row` that contains the following widgets:
+
+<ul markdown="1">
+  <li markdown="1">
+  An `Icon` widget set to `Icons.account_circle` with a size of 50 pixels.
+  </li>
+  <li markdown="1">
+  A `Padding` widget that creates a space of 8 pixels around the `Icon` widget. 
+
+  To do this, you can specify `const EdgeInsets.all(8.0)` for the padding property.
+
+  For example: 
+  </li>
+</ul>
+
+  ```dart
+  Padding(
+            padding: const EdgeInsets.all(8.0),
+            child: Icon(Icons.account_circle, size: 50)),
+  ```
+{{site.alert.end}}
+<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=95dcc1451aea8412669c41eb8a1a5f23&amp;theme=dark&amp;split=60" width="100%" height="400px"></iframe>
+
+### Part 2
+{:.no_toc}
+
+This part also contains two examples, both focusing on contact information. 
+In the first example, you'll wrap the `Row` you built in a `Column`, 
+so you can create space for the contact information. 
+In the second example, you'll enter the contact information. 
+At the end of Part 2, your code will contain a `Column` within a `Row` within a `Column`.
+
+#### Exercise: Tweak the layout
+{:.no_toc}
+{{site.alert.secondary}}
+  
+  Wrap your `Row` inside a `Column` that contains the following widgets:
+
+<ul markdown="1">
+  <li markdown="1"> 
+  A `mainAxisSize` property set to `MainAxisSize.min`. 
+  </li>
+  <li markdown="1">
+  A `crossAxisAlignment` property set to `CrossAxisAlignment.stretch`.
+  </li>
+  <li markdown="1">
+  A `SizedBox` widget with a height of 8.
+  </li>
+  <li markdown="1">
+  An empty `Row` class where you'll add your contact information. 
+  </li>
+  <li markdown="1">
+  A second `SizedBox` widget with a height of 16.
+  </li>
+  <li markdown="1">
+  A second empty `Row` class where you'll add different icons (covered in [Part 3](#part-3)). 
+  </li>
+</ul>
+
+  Your `SizedBox` widgets and empty `Row` classes should be formatted like this: 
+
+  ```dart
+
+     ],
+    ),
+    SizedBox(),
+    Row(), // First empty Row class.
+    SizedBox(),
+    Row(), // Second empty Row class.
+   ],
+  );
+
+  ```
+{{site.alert.end}}
+
+<iframe src="https://dartpad.dev/experimental/embed-new-flutter.html?id=c3ac34ed8952724a0ecb0af1445c2af8&amp;theme=dark&amp;split=60" width="100%" height="400px"></iframe>
+
+#### Example: Enter contact information 
+{:.no_toc}
+{{site.alert.secondary}}
+  Enter the following widgets inside the first empty `Row` class: 
```

Comment:
class -> widget

I'm not going to mark these from here on, but do a global search for "class" to make sure you're using it correctly. It's easy to slip up and say "class" when you mean object/instance/widget.